packages/ove-service-layout/README.md

```diff
@@ -0,0 +1,69 @@
+# OVE Layout Service
+
+The Layout Service can be used in addition to the OVE to support more complex layouts,
+other than the absolute coordinates system.
+
+## Build
+
+This service is a standard npm build, which can be either build with lerna (as per instructions)
+in the Readme file in the root of this repository.
+
+Alternatively, it can be build and run with standard npm, using the package.json.
+
+## Testing & developing
+
+A standard working example is provided in the [Working Example](docs/WORKING_EXAMPLE.md) document.
+
+Testing this service requires either a working OVE instance or a mocked service. The mocked service
+is available as part of this project. To run it:
+
+```bash
+npm run mockup
+```
+
+This will spin up a json mockup rest server, exposing the following url: [http://localhost:3004/space-info](http://localhost:3004/space-info).
+
+## Available layouts
+
+The layouts and parameters are available in the [layouts](docs/LAYOUTS.md) document.
+
+## Unit tests
+
+```bash
+> npm run coverage
+ 
+ [PASS]  src/layout/manager.test.js
```

Comment:
@oserban, why can't your tests always be found under the src/test/ folder?
I know this is a very Java-like question, but I think it makes things a bit more organised. WDYT?

Reply:
The jest/react standard way is to keep the test close to the source with a different naming convention. I just followed the guidelines here :-)